testing/src/cypress/page-objects/Institution-objects/ManageLocationObject.ts

```diff
@@ -87,6 +91,78 @@ export default class ManageLocationObject {
     return cy.contains("Country is required");
   }
 
+  countryDropDownMenu() {
+    return cy.get("[data-cy='selectedCountry']").parent();
+  }
+
+  countryCanadaFromDropDownMenu() {
+    return cy.get('div[data-value="Canada"]');
+  }
+
+  countryOtherFromDropDownMenu() {
+    return cy.get('div[data-value="other"]');
+  }
+
+  otherCountryInputText() {
+    return cy.get("[data-cy='otherCountry']");
+  }
+
+  provinceDropDownMenu() {
+    return cy.get("[data-cy='provinceState']").parent();
+  }
+
+  provinceABFromDropDown() {
```

Comment:
From provinces, we could avoid some repetition and build a method to accept the province as an argument.
```ts
provinceFromDropDown(province: Provinces) {
    return cy.get(`div[data-value="${province}"]`);
}
```

Reply:
added new function